public/pages/Transforms/utils/metadataHelper.tsx

```diff
@@ -90,10 +90,11 @@ export const selectInterval = (
   intervalTimeunit: string,
   intervalError: string,
   onChangeInterval: (e: ChangeEvent<HTMLInputElement>) => void,
-  onChangeTimeunit: (value: ChangeEvent<HTMLSelectElement>) => void
+  onChangeTimeunit: (value: ChangeEvent<HTMLSelectElement>) => void,
+  useUpdatedUX?: boolean
 ) => (
   <React.Fragment>
-    <EuiFlexGroup style={{ maxWidth: 400 }}>
+    <EuiFlexGroup style={{ maxWidth: 400 }} alignItems={useUpdatedUX ? "center" : undefined}>
```

Comment:
Its being generally discouraged by tiger team to change look and feel of individual component based on feature flag state. I'd assume the old UI should also be needing some alignment value? can we have screenshot of old and new attached here once to see comparison of alignment

Reply:
I think we can remove it for now